scripts/copy_scripts.sh

```diff
@@ -4,4 +4,13 @@ cp ../scripts/entry.sh .
 if [[ ! -f "Dockerfile" ]];
 then
     cp ../scripts/Dockerfile .
+fi
+
+folders=("public" "routes" "uploads" "utils" "view");
+if [[ ${PWD##*/} == "fastify3" ]];
```

Comment:
Instead of creating another script to do this and adding it to the workflow, I figured it could just be included in the existing one to minimize other changes